libraries/stdlib/genosl/mx_ap1_to_rec709_color3.osl

```diff
@@ -1,7 +1,7 @@
 void mx_ap1_to_rec709_color3(color _in, output color result)
 {
     vector4 outColor = vector4(_in[0], _in[1], _in[2], 0.);
-    matrix m = transpose(matrix(1.705079555511475, -0.1297005265951157, -0.02416634373366833, 0., -0.6242334842681885, 1.138468623161316, -0.1246141716837883, 0., -0.0808461606502533, -0.008768022060394287, 1.148780584335327, 0., 0., 0., 0., 1.));
+    matrix m = matrix(1.705079555511475, -0.1297005265951157, -0.02416634373366833, 0., -0.6242334842681885, 1.138468623161316, -0.1246141716837883, 0., -0.0808461606502533, -0.008768022060394287, 1.148780584335327, 0., 0., 0., 0., 1.);
```

Comment:
Hi @jstone-lucasfilm ,

Believe the recommendation here from @rasmusbonnedal is to make this 3x3 matrix and call the vec3 version of transform() which is built in.

As far as I know vec4 (alpha) is only used for a few output (display) transforms and no input transforms, which is the same as alpha=0 in a vec4 computation.